Code/Framework/AzCore/AzCore/Memory/HeapSchema.h

```diff
@@ -32,17 +32,11 @@ namespace AZ
          */
         struct Descriptor
         {
-            Descriptor()
-                : m_numMemoryBlocks(0)
-                , m_isMultithreadAlloc(true)
-            {}
-
-            static const int        m_memoryBlockAlignment = 64 * 1024;
             static const int        m_maxNumBlocks = 5;
-            int                     m_numMemoryBlocks;                          ///< Number of memory blocks to use.
-            void*                   m_memoryBlocks[m_maxNumBlocks];             ///< Pointers to provided memory blocks or NULL if you want the system to allocate them for you with the System Allocator.
-            size_t                  m_memoryBlocksByteSize[m_maxNumBlocks];     ///< Sizes of different memory blocks, if m_memoryBlock is 0 the block will be allocated for you with the System Allocator.
-            bool                    m_isMultithreadAlloc;                       ///< Set to true to enable multi threading safe allocation.
+            int                     m_numMemoryBlocks = 1;                      ///< Number of memory blocks to use.
+            void*                   m_memoryBlocks[m_maxNumBlocks] = {};        ///< Pointers to provided memory blocks or NULL if you want the system to allocate them for you with the System Allocator.
+            size_t                  m_memoryBlocksByteSize[m_maxNumBlocks] = {4 * 1024}; ///< Sizes of different memory blocks, if m_memoryBlock is 0 the block will be allocated for you with the System Allocator.
+            bool                    m_isMultithreadAlloc = true;                ///< Set to true to enable multi threading safe allocation.
```

Comment:
This seems like it has limited use if it is hardcoded to 5 blocks.

Is even useful to have more than 1 block, since the default number of memory blocks is 1?

Reply:
TBH, this is not practical at all, will put it in the list of things to cleanup